client/asset/dcr/dcr.go

```diff
@@ -2788,13 +2817,17 @@ func (dcr *ExchangeWallet) fatalFindRedemptionsError(err error, contractOutpoint
 
 // Refund refunds a contract. This can only be used after the time lock has
 // expired. This MUST return an asset.CoinNotFoundError error if the coin is
-// spent.
+// spent. If the provided fee rate is zero, an internal estimate will be used,
+// otherwise it will be used directly.
 // NOTE: The contract cannot be retrieved from the unspent coin info as the
 // wallet does not store it, even though it was known when the init transaction
 // was created. The client should store this information for persistence across
 // sessions.
-func (dcr *ExchangeWallet) Refund(coinID, contract dex.Bytes, feeSuggestion uint64) (dex.Bytes, error) {
-	msgTx, err := dcr.refundTx(coinID, contract, 0, nil, feeSuggestion)
+func (dcr *ExchangeWallet) Refund(coinID, contract dex.Bytes, feeRate uint64) (dex.Bytes, error) {
+	if feeRate == 0 {
+		feeRate = dcr.targetFeeRateWithFallback(2, 0)
```

Comment:
I had to really work through this. The way the `feeRate` is generated in **core**, if `feeRate == 0` here, that means that `(*ExchangeWallet).FeeRate` just failed (as well as any attempt to get a rate from the server or the order book). So this is likely to fail again, and we probably end up with the `fallbackFeeRate`. Is that your read too?

Reply:
Yeah, with core's current use pattern, that's likely to happen.  I thought about just `dcr.feeRateWithFallback(feeRate)` here, but figured what the heck, might as well try an internal rate assuming we don't know what caller is doing.

Reply:
Your call. Even just a comment.